### PR TITLE
feat/add_next_and_previous_tasks_endpoint

### PR DESCRIPTION
**Pull request by spiridonovfed**

### CODE REVIEW

File: annotation/app/tasks/services.py [hunk 1]
```diff
@@ -651,3 +652,29 @@ def save_agreement_metrics(
...
+) -> int:
+    list_of_task_ids = [task.id for task in tasks_list]
+    return list_of_task_ids.index(task_id)
+
+
+def get_previous_task(
+    tasks_list: List[ManualAnnotationTaskSchema], current_index: int
+) -> Optional[ManualAnnotationTaskSchema]:
+    previous_task_index = current_index - 1
+    if previous_task_index < 0:
```
Comment (khyurri) [inline annotation/app/tasks/services.py:668]: Let's simplify: 
If we order by id, so previous task is:
`select .. where id < current_id order by id desc limit 1`
and next is
`select ... where id > current_id order by id asc limit 1`

**Review by Andrka — COMMENTED** [review]
Could you add some simple tests?

File: annotation/app/tasks/services.py (hunk 1 above)
Comment (spiridonovfed) [reply annotation/app/tasks/services.py]: Rewritten in a suggested way

**Comment by spiridonovfed** [comment]
> Could you add some simple tests?

Sure. Tests are added

**Review by Andrka — APPROVED** [review]

**Pull request merged**